sorc/ncep_post.fd/CALRAD_WCLOUD_newcrtm.f

```diff
@@ -116,14 +116,12 @@ SUBROUTINE CALRAD_WCLOUD
   !      integer,parameter::  n_clouds = 4 
   integer,parameter::  n_aerosols = 0
   ! Add your sensors here
-  integer(i_kind),parameter:: n_sensors=23
+  integer(i_kind),parameter:: n_sensors=21
   character(len=20),parameter,dimension(1:n_sensors):: sensorlist= &
```

Comment:
@gspetro-NOAA Could you add a change log entry with your name in this subroutine?